charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -529,6 +529,12 @@ admissionWebhooks:
   ## Secret labels
   secretLabels: { }
 
+## Agent workloads list
+agentWorkloads:
```

Comment:
is `agentWorkloads` going to be the new `agent` in the eks addon advanced config? Would we break customers if we change that?